packages/vscode-apollo/create-server-symlink.js

```diff
@@ -0,0 +1,9 @@
+"use strict";
+const symlinkDir = require("symlink-dir");
+const path = require("path");
+
+symlinkDir("../../apollo-language-server", "./node_modules/")
```

Comment:
Is it ok that this is an async function? Seems this could be fragile if not synchronous.

Reply:
I'm not 100% sure. I haven't seen it error _yet_ because of it though 😬

Reply:
@trevor-scheer Your concern here is very reasonable.  If any other instruction were after this, and not within the `then`-able, this would need to be `await`ed.  This works because:

1. There is nothing after the call to `symlinkDir`.
2. This is a stand-alone module invoked directly by `node` (That is to say, it's not traversed _via_ another file/module but directly by the interpreter).
3. The Node.js interpreter (`node`) evaluates this set of instructions (which populates the the event loop) and then proceeds to process the event loop until it's empty before exiting (`exit()`).